home-open-api/src/main/resources/yaml/openapi.yaml

```diff
@@ -1456,21 +1466,21 @@ components:
       properties:
         title:
           type: string
-          example: News In The Local Area
+          example: "News In The Local Area"
```

Comment:
Honestly, I think that quotes here bring any visibility neither readability. Considering yaml specification they are redundant and I suggest removing them.

Reply:
I’m not so sure. In my experience, and other team members also agree that examples taken in quotation marks are marked in green (depends on the editor), they are visually easy to search for. Without quotes, they merge with the general text. I suggest leaving the quotes for the examples. We can also bring this question for discussion to the next meeting.